tutor/commands/local.py

```diff
@@ -31,14 +32,14 @@ def __init__(self, root: str, config: Config):
 
 # pylint: disable=too-few-public-methods
 class LocalContext(compose.BaseComposeContext):
-    def job_runner(self, config: Config) -> LocalJobRunner:
-        return LocalJobRunner(self.root, config)
+    def job_runner(self) -> LocalJobRunner:
+        return LocalJobRunner(self.root, self.config)
 
 
 @click.group(help="Run Open edX locally with docker-compose")
 @click.pass_context
 def local(context: click.Context) -> None:
-    context.obj = LocalContext(context.obj.root)
+    context.obj = LocalContext(context.obj.root, tutor_config.load(context.obj.root))
```

Comment:
This line will fail the first time someone runs `tutor local quickstart` with an empty project root.